File: Data Structures & Algorithms/missing-number/submission-0.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int sum = n*(n+1)/2;
        int x = 0;
        for (auto& num: nums) {
            x += num;
        }
        return sum - x;
    }
};

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        n = len(nums)
        dp = [False] * n
        dp[n-1] = True
        for i in range(n-2, -1, -1):
            print(i)
            for d in range(nums[i] + 1):
                print(i, d)
                dp[i] = dp[i] or dp[i + d]
                print(dp[i])
        return dp[0]